### DISCUSSION TITLE
clickable / linked relationship

### DISCUSSION THREAD

**Discussion by Dontorpedo** [filamentphp/filament#4382] [Ideas]
I need some Fields to be linked to the corresponding relationship eg.:

In my house resource form i show the customer_id, which should be linked to the owner of the house, so i can move fast between those two resources.. 

or invoice house_id linked to the house resource and so on.. 

how can i do this in filament?

**Comment by danharrin** [comment]
You could add something like `->hint(fn ($state): string => filled($state) ? '[Open](' . CustomerResource::getUrl('edit', ['record' => $state])  . ')' : null)`

This is a markdown link that appears when the field is filled, and allows you to visit the Edit page on the CustomerResource
